.github/workflows/ci.yml

```diff
@@ -62,6 +62,12 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN clean verify -B --strict-checksums -V -T C1 -DskipTests -P ci -pl '!:trino-server-rpm'
+      - name: Check if all commits compile
+        run: |
+          git fetch origin
+          git config user.name "$(git --no-pager log --format=format:'%an' -n 1)"
```

Comment:
Why?

Reply:
Git rebase needs an identity to assign to commits. Here I'm using the identity from the top commit in the PR.

Reply:
Provide any random identity - doesn't matter since we won't use the results.